generic/tests/netperf.py

```diff
@@ -95,28 +95,41 @@ def mtu_set(mtu):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    try:
-        vm.wait_for_serial_login(timeout=login_timeout, restart_network=True).close()
-    except virt_vm.VMIPAddressMissingError:
-        pass
-
-    if len(params.get("nics", "").split()) > 1:
-        session = vm.wait_for_login(nic_index=1, timeout=login_timeout)
-    else:
-        session = vm.wait_for_login(timeout=login_timeout)
-
+    expect_status = params['config_cmds_expect_status']
```

Comment:
You don't really use it now, shall we remove? Thanks

Reply:
ok,  I have update.